tests/php/mocks/CartMock.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\Tests\Mocks;
+
+use Automattic\WooCommerce\Blocks\BlockTypes\Cart;
+use Automattic\WooCommerce\Blocks\Package;
+use Automattic\WooCommerce\Blocks\Assets\Api;
+use Automattic\WooCommerce\Blocks\Assets\AssetDataRegistry;
+use Automattic\WooCommerce\Blocks\Integrations\IntegrationRegistry;
+
+/**
+ * CartMock used to test cart block functions.
+ */
+class CartMock extends Cart {
+
+	/**
+	 * We initaite our mock class.
+	 */
+	public function __construct() {
+		parent::__construct(
+			Package::container()->get( API::class ),
+			Package::container()->get( AssetDataRegistry::class ),
+			new IntegrationRegistry(),
```

Comment:
`IntegrationRegistry` Isn't part of the DI, a new instance of it is passed to each class instance.